sdk/keyvault/Directory.Build.props

```
@@ -1,6 +1,6 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup Condition="$(MSBuildProjectName.EndsWith('.Samples'))">
-    <IsSampleProject>true</IsSampleProject>
+  <PropertyGroup Condition="$(MSBuildProjectName.EndsWith('.Samples'))And!$(MSBuildProjectName.StartsWith('Azure.ResourceManager'))">
```

Not sure this even works, but please add spaces for readability/maintainability.
```suggestion
  <PropertyGroup Condition="$(MSBuildProjectName.EndsWith('.Samples')) And !$(MSBuildProjectName.StartsWith('Azure.ResourceManager'))">
```